app/views/dishes/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Dish: <%= @dish.name %></h1>
+<h2><%= @dish.description %></h2>
+<h3>Chef: </h3><%= @dish.chef.name %>
+<% total_calories = 0 %>
+<h3>Ingredients: </h3><% @dish.ingredients.each do |ingredient| %>
+    <%= ingredient.name %><br>
+    <% total_calories += ingredient.calories %>
+<% end %>
+<h3>Total Calories: </h3><%= total_calories %>
```

Comment:
The way you are getting the total calories is incorrect. You are using Ruby to process database data over using pure Active Record. This code should be using all AR and be abstracted to a model method.